CHANGELOG.md

```diff
@@ -22,7 +22,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Fixed
 
-- Nothing.
+- [#225](https://github.com/zendframework/zend-mail/pull/225) fixes duplicate `to` and `subject` headers on Windows
```

Comment:
add link to previous issue too: https://github.com/zendframework/zend-mail/issues/22

Reply:
Done